.prettierignore

```diff
@@ -29,3 +29,6 @@ core/tests/wdio.conf.js
 core/tests/data/dummy_ci_tests/dummy_webdriverio.conf.js
 scripts/linters/test_files
 core/tests/services_sources
+
+# Exclude JSON files in CI test suite config
+!core/tests/ci-test-suite-configs/*.json
```

Comment:
why are we excluding CI config JSON files? How does prettier want to reformat them?

Reply:
@U8NWXD Current JSON format in `acceptace.json` and `e2e.json` is same as what prettier autoformats. Currently, we are required to manually format, but excluding this files from ignore list means it can be formatted easily.

Reply:
I'm confused. If the format that prettier generates is the same as the current format, then I would expect excluding the files from prettier formatting to make no difference. Why does adding these files to the ignore list make formatting them easier?

Reply:
Sorry, I didn't get it properly.

Do you mean we should have prettier format these files? If so, I am not excluding these files from prettier. They were excluded by `*.json` in line 21, and I am re-including using `!` (`!core/tests/ci-test-suite-configs/*.json`).

Should I change comment to `# Include JSON files in...`

Reply:
Ahh okay, I got confused by the comment. I agree with the changes you're making here. Just update the comment to clarify that we're instructing prettier to format the suite configs

Reply:
Done.